region/maridia/inner-yellow.json

```diff
@@ -1020,7 +1039,8 @@
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
-                    "canSpringBallJumpMidAir"
+                    "canSpringBallJumpMidAir",
```

Comment:
max height springball jumps have been called `trickyJump` too.

Reply:
I don't really think it is as tight as some of the others, it just took me a couple tries. I could have gotten lucky, but I was thinking maybe it's not quite a max height one?

Reply:
Looks like a max height jump to me.  You can actually use the choot for a slightly less precise jump.